lib/vet360/configuration.rb

```diff
@@ -15,7 +15,7 @@ def connection
 
         # @TODO Remove for production...
         # (This is really helpful for setting up cassettes)
-        # faraday.response :logger, ::Logger.new(STDOUT), bodies: true
+        faraday.response :logger, ::Logger.new(STDOUT), bodies: true
```

Comment:
Reminder to comment this back out before shipping.  As I understand it, this would log personal info that we don't want logged.